pkg/ostree/client.js

```diff
@@ -388,6 +388,24 @@ function RPMOSTreeDBusClient() {
         return proxy;
     };
 
+    self.signature_obj = function(signature) {
+        if (!signature.v)
+            return;
+
+        var by = signature.v[11];
```

Comment:
Unfortunately this is not documented, and internal to the ostree library. The dbus variant type is a very opaque ```av```. I can see what our options are on the dbus side, but since this hasn't changed since it was initially implemented I figured we'd use what we have and hopefully get something more stable in later iterations.